### PR TITLE
spidermultusconfig: ovs cni config supported

### PR DESCRIPTION
**Pull request by cyclinder**
---
name: Pull request
about: Tell us about your contribution
labels: ["pr/release/none-required"]
---

---
## Thanks for contributing !

Before submitting a pull request, make sure you read about our Contribution notice here: <https://spidernet-io.github.io/spiderpool/contributing/development/pullrequest/#need-review>

**What this PR does / why we need it**:

- spidermultusconfig: ovs cni config supported, Now we can create multus NAD for CNI of type ovs via spidermultusconfig
- remove unused fields of  coordinator and ifacer cni  config

**Which issue(s) this PR fixes**:

https://github.com/spidernet-io/spiderpool/issues/2121
https://github.com/spidernet-io/spiderpool/issues/2006

**Special notes for your reviewer**:

**make sure your commit is signed off**

### CODE REVIEW

File: pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go [hunk 1]
```diff
@@ -59,7 +62,7 @@ type SpiderMacvlanCniConfig struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=4094
-	VlanID *int32 `json:"vlanID,omitempty"`
```
Comment (weizhoublue) [inline pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:62]: 这是一个不兼容的 CRD 变更， 有很多 一些 其它 存量的 doc 都需要 纠正

Comment (cyclinder) [reply pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:62]: 可以改下文档，这个字段可以和spiderpool ippool 的vlan保持一致

File: charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml [hunk 2]
```diff
@@ -168,14 +169,57 @@ spec:
...
+                      ipv4:
+                        items:
+                          type: string
+                        type: array
+                      ipv6:
+                        items:
+                          type: string
+                        type: array
+                    type: object
+                  trunk:
```
Comment (weizhoublue) [inline charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml:200]: 这个是否不暴露

File: charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml [hunk 3]
```diff
@@ -126,7 +127,7 @@ spec:
                     items:
                       type: string
                     type: array
-                  vlanID:
+                  vlan:
```
[the remark this answers is not in the document]
Comment (cyclinder) [reply charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml:130]: 不是一定要改，当然能改就最好改了，现在不改等这个版发了就不好改了

File: charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml (hunk 2 above)
Comment (cyclinder) [reply charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml:200]: 不暴露... 有风险？

File: charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (weizhoublue) [reply charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml:130]: 0.6.0 的开源用户怎么办

[the remark this answers is not in the document]
Comment (cyclinder) [reply charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml:130]: 那好吧 -.- 我真的很想改过来，以后是不是都没机会改了

File: charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml (hunk 2 above)
Comment (cyclinder) [reply charts/spiderpool/crds/spiderpool.spidernet.io_spidermultusconfigs.yaml:200]: 这个还是保留吧，这只是原生ovs-cni的能力

**Pull request merged**